docs-src/concepts/what-is-asynchronous-activity-completion.md

```diff
@@ -19,11 +19,19 @@ The Temporal Client can then be used to both Heartbeat Activity Execution progre
 
 #### When to use Async Completion
 
-The intended use-case for this feature is when an external system has the final result of a computation, started by an Activity.
+The intended use case for this feature is when an external system has the final result of a computation, started by an Activity.
+However, often it works better for the Activity to notify the external system, complete normally, and then wait in the Workflow for a [Signal](/concepts/what-is-a-signal) from the external system.
 
-Consider using Asynchronous Activities instead of Signals if the external process is unreliable and might fail to send critical status updates through a Signal.
+Consider using Async Completion when
```

Comment:
@djmagee I think no colon is what you said to do when I want to avoid "in the following scenarios:"?

Reply:
(which feels wordy to me)

Reply:
Right, per https://developers.google.com/style/colons#introductory-phrase-preceding-colon.